pkg/apps/srv/backstage/install.yaml

```diff
@@ -90,15 +78,93 @@ spec:
       labels:
         app: backstage
     spec:
+      serviceAccountName: backstage
+      volumes:
+        - name: backstage-config
+          projected:
+            sources:
+              - configMap:
+                  items:
+                    - key: app-config.yaml
+                      path: app-config.yaml
+                  name: backstage-config
       containers:
         - name: backstage
+          command:
+            - node
+            - packages/backend
+            - --config
+            - config/app-config.yaml
           image: public.ecr.aws/cnoe-io/backstage:v0.0.3
           imagePullPolicy: IfNotPresent
           ports:
             - name: http
               containerPort: 7007
-          envFrom:
-            - secretRef:
-                name: postgres-secrets
-            - secretRef:
-                name: backstage-secrets
\ No newline at end of file
+          volumeMounts:
+            - mountPath: /app/config
+              name: backstage-config
+              readOnly: true
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/name: backstage
+  name: backstage-config
+  namespace: backstage
+data:
+  app-config.yaml: |
+    app:
+      title: CNOE
+      baseUrl: http://localhost:3000
+    organization:
+      name: CNOE
+    backend:
+      baseUrl: http://localhost:7007
+      listen:
+        port: 7007
+      csp:
+        connect-src: ["'self'", 'http:', 'https:']
+      cors:
+        origin: http://localhost:3000
+        methods: [GET, HEAD, PATCH, POST, PUT, DELETE]
+        credentials: true
+      database:
+        client: better-sqlite3
+        connection: ':memory:'
+      cache:
+        store: memory
+    integrations: {}
+    proxy:
+    techdocs:
+      builder: 'local'
+      generator:
+        runIn: 'docker'
+      publisher:
+        type: 'local'
+    auth:
+      environment: local
+      providers: {}
+    scaffolder: {}
+    catalog:
+      import:
+        entityFilename: catalog-info.yaml
+        pullRequestBranchName: backstage-integration
+      rules:
+        - allow: [Component, System, API, Resource, Location, Template]
```

Comment:
is there a reason why are we not allowing domains into backstage?

Reply:
Good point, really no reason to prohibit it for local developments.

Reply:
Although in general I could see us wanting to allow customization on things like `entityFilename` where some organizations use custom file names so I wonder if some of this being done through a GoLang template may make sense in the future.

Reply:
Yea we can allow for more customization in the future. These manifests allow you to get started quickly locally.